presenters/thread.rb

```diff
@@ -76,7 +76,7 @@ def to_hash with_responses=false, resp_skip=0, resp_limit=nil, recursive=true
   #     children, if recursive is true)
   #   response_count
   #     The total number of responses
-  def get_paged_merged_responses(thread_id, responses, skip, limit, recursive=false)
+  def get_paged_merged_responses(thread_id, responses, skip, limit, recursive=true)
```

Comment:
Can you explain why this default changed and its implications?  It seems like some test should have to change to account for it.